config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Encass.java

```diff
@@ -156,4 +162,18 @@ public void sortArgumentsAndResults() {
         setArguments(getArguments().stream().collect(toCollection(() -> new TreeSet<>(Comparator.comparing(EncassArgument::getName)))));
         setResults(getResults().stream().collect(toCollection(() -> new TreeSet<>(Comparator.comparing(EncassResult::getName)))));
     }
+
+    @Override
+    public boolean hasBundleAnnotation(){
+        if(hasAnnotated()){
+            return annotations.stream().anyMatch(annotation -> ANNOTATION_TYPE_BUNDLE.equals(annotation.getType()));
```

Comment:
This logic can be changed to: `annotations.contains(new Annotation(ANNOTATION_TYPE_BUNDLE));` as it is a `Set` and `Annotation.equals()` look only for type. Will reduce the complexity to constant time O(1).